Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -166,7 +166,9 @@ file that was distributed with this source code.
         {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
             {{- widget|raw -}}
-            {{- label is not same as(false) ? label|trans({}, translation_domain) -}}
+            {%- if sonata_admin.options['use_icheck'] -%}&nbsp;{%- endif -%}
```

Comment:
If the `label` is wrapper in a `div.checkbox` then the label won't be bold.

With just your CSS fix :

![css fix](https://cloud.githubusercontent.com/assets/2057992/13984332/c6a81d00-f0f5-11e5-9209-3078450b7fbe.PNG)


When wrapping with `div.checkbox` + my css fix : 

![bootstrap fix](https://cloud.githubusercontent.com/assets/2057992/13984344/d9c6cd96-f0f5-11e5-83f5-0a718b7a1a40.PNG)